.github/workflows/e2e_tests_cypress.yml

```diff
@@ -4,12 +4,12 @@ on:
     branches:
       - master
       - release_3_4
-      - refactor
+      - tests_load_sql
```

Comment:
Except if you are running on your local git repo, but any pull request against `master` or `release_3_4` will run the test.
`refactor` was special because it wasn't a PR for many months.

Reply:
Yes, I'll change that and push force when my work is finished. I should have create a draft :)

Reply:
@Gustry I have also study different ways to launch this workflow less frequently. Still thinking...